packages/contentful/src/crn/queries/teams.queries.ts

```diff
@@ -23,6 +23,38 @@ export const teamsContentQueryFragment = gql`
         url
       }
     }
+    linkedFrom {
+      teamMembershipCollection(limit: 100) {
```

Comment:
This is sufficient based on current data (the largest team is ~50-60 users) but we may want to add an additional check so that if the users on a team exceeds 100 then it's users are aggregated across multiple queries.

Reply:
meh :D